index.js

```diff
@@ -10,12 +10,15 @@ var licenseDir = __dirname + "/license-files/"
 // Alternate abbreviations used by package.json files.
 var licenseAliases = {
     "BSD": "BSD-2-Clause",
-    "MIT/X11": "MIT"
+    "MIT/X11": "MIT",
+    "apache version 2.0": "Apache-2.0"
 }
 
 var licenses = []
 function normalizeText(text) {
-    return text.replace(/[^a-z0-9\s]/ig, '').toLowerCase().trim().split(/[\s\n]+/).join(' ')
+    // Normalize text for matching purposes.
+    // Consider "-" same as space: Apache-2.0 -> apache 20, Apache 2.0 -> apache 20
+    return text.replace(/[^a-z0-9\s-]/ig, '').toLowerCase().trim().split(/[\s\n-]+/).join(' ')
```

Comment:
Wonder if it makes sense to do the toLowerCase first, and the split/join with regexp